presto-docs/src/main/sphinx/sql/show-create-table.rst

```diff
@@ -0,0 +1,21 @@
+=================
+SHOW CREATE TABLE
+=================
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW CREATE TABLE table_name
+
+Description
+-----------
+
+Display the column names and types followed by table properties.
+
+Examples
+--------
+Show create table for ``orders``::
```

Comment:
This example would be a lot more useful with sample output. See the SELECT documentation for an example of how to do that.